--- a/debian/rules       2014-02-18 03:13:28.000000000 +0000
+++ b/debian/rules       2016-04-04 02:03:23.732296279 +0000
@@ -41,8 +41,10 @@
		--enable-icmp \
		--enable-zph-qos \
		--enable-ecap \
+		--enable-ssl-crtd \
		--disable-translation \
		--with-swapdir=/var/spool/squid \
+		--with-openssl \
		--with-logdir=/var/log/squid \
		--with-pidfile=/var/run/squid.pid \
		--with-filedescriptors=65536 \
